packages/modals/src/elements/Modal.tsx

```diff
@@ -138,17 +141,32 @@ export const Modal = React.forwardRef<HTMLDivElement, IModalProps>(
           bodyElement.style.paddingRight = `${bodyPaddingRight + scrollbarSize}px`;
         }
 
-        const previousBodyOverflow = bodyElement.style.overflow;
+        previousBodyOverflow = bodyElement.style.overflow;
 
         bodyElement.style.overflow = 'hidden';
```

Comment:
Why is `body` overflow management being retained?

Reply:
Oh wow, it doesn't and the solution can be simplified a lot. Thanks!